.github/workflows/upload.yaml

```
@@ -62,8 +54,15 @@ jobs:
           upload_all_metadata \
           --configfile profiles/upload.yaml
 
-  trigger:
+  run-nextclade:
     needs: [upload]
+    permissions:
+      id-token: write
+    uses: nextstrain/seasonal-flu/.github/workflows/run-nextclade.yaml
```

I omitted it in Slack with my suggestion, but a ref is required for `uses:`, e.g.

```suggestion
    uses: nextstrain/seasonal-flu/.github/workflows/run-nextclade.yaml@master
```

But upon further consideration, since the workflows exist in the same repo as each other, it'd really be best to use

```suggestion
    uses: ./.github/workflows/run-nextclade.yaml
```

so that `run-nextclade.yaml` is used at the same revision that this calling workflow is using (see [docs](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#calling-a-reusable-workflow)).

---

> ...since the workflows exist in the same repo as each other, it'd really be best to use...

Thanks! I prefer the second option, too.